README.md

```diff
@@ -205,6 +205,10 @@ impl<Chain: CwEnv> Example<Chain> {
 }
 ```
 
+### Testing with OsmosisTestTube
+
+OsmosisTestTube is available for testing in cw-orchestrator. In order to use it, you may need to install [clang] and [go] to compile the osmosis blockchain that serves as the backend for this env. 
```

Comment:
Is the compilation done by the crate or is there other setup? A link to the test-tube repo would be great here as well.